projects/plugins/crm/src/automation/class-automation-workflow.php

```diff
@@ -21,7 +21,23 @@ class Automation_Workflow {
 	 * @since $$next-version$$
 	 * @var int|string
 	 */
-	private $id;
+	protected $id;
```

Comment:
Technically this change is unnecessary for this PR, but it unifies how we set property visibility.